tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -4138,6 +4138,32 @@ def _assert_is_fit(self, message_suffix: str = None):
                 error_message = f"{error_message} `.{message_suffix}`."
             raise AssertionError(error_message)
 
+    def get_conformal_predictions(self, test_data, model: str) -> (List[np.ndarray], List[np.ndarray], List[np.ndarray]):
```

Comment:
The name here might be misleading, as this method does not strictly returns "conformal predictions", but it returns ensembles of outputs and targets used in the conformal method.